src/components/appShell/AppShell.cy.tsx

```diff
@@ -0,0 +1,8 @@
+import { AppShell } from './AppShell';
+
+describe('AppShell', () => {
+  it('should work', () => {
+    cy.mount(<AppShell content="" />);
+    cy.viewport(1280, 720);
+  });
+});
```

Comment:
a test that we can have is testing the app shell items (like does clicking on them sets them in active mode? or clicking on one while another is active would make the other non active?)

Reply:
I checked it 
I didn't get what u mean